prod-aws/kafka-shared/iam_credentials.tf

```diff
@@ -21,3 +21,12 @@ module "iam_credentials_producer" {
 
   cert_common_name = "auth-customer/credentials-api"
 }
+
+module "iam_credentials_consumer" {
+  source = "../../modules/consumer"
+
+  topic          = kafka_topic.iam_credential_v1.name
```

Comment:
should it be `kafka_topic.iam_credentials_v1.name`?

Reply:
good spot!